src/abstractGraphQLClient.ts

```diff
@@ -1,29 +1,13 @@
 import { GraphQLClient } from 'graphql-request';
 import * as Dom from 'graphql-request/dist/types.dom';
-import { URL } from 'url';
 import { Options } from './abstractClient';
-import path from 'path';
-import { GetProductsType } from './catalog';
-
-export type GraphQLResponseTypes = GetProductsType;
-
-export abstract class AbstractGraphQLClient {
-  /**
-   * Base path for HTTP calls
-   */
-  protected basePath = '';
-
-  /**
-   * Current path for HTTP calls
-   */
-  protected path = '';
+import * as path from 'path';
```

Comment:
Warning:

Project using src/abstractGraphQLClient.ts could have issues importing path.
To resolve it, I changed the config of webpack like in this PR
--> https://stash.arrowcloud.org/projects/AAS/repos/xsp-web/pull-requests/8/diff#vue.config.js